Makefile

```diff
@@ -440,6 +440,11 @@ build/toolchain/bin/golangci-lint$(EXE_EXTENSION):
 	mv $(TOOLCHAIN_DIR)/temp-golangci/golangci-lint$(EXE_EXTENSION) $(TOOLCHAIN_BIN)/golangci-lint$(EXE_EXTENSION)
 	rm -rf $(TOOLCHAIN_DIR)/temp-golangci/
 
+build/toolchain/python/:
+	# mkdir -p $(TOOLCHAIN_DIR)/python/
+	if ! [ -d "$(TOOLCHAIN_DIR)/python" ]; then virtualenv --python=python3 $(TOOLCHAIN_DIR)/python/; fi
```

Comment:
Why is there an if statement here then? If this rule is configured correctly it should only execute if the directory does not exist.

Reply:
Done.